Dockerfile

```diff
@@ -1,3 +1,11 @@
+FROM golang:1.10 as tester
+LABEL tester=true
```

Comment:
Don't need these labels, we only use them in OLM because we're pulling out different stages to tag separately